bitbox-bridge/release/darwin/package.sh

```diff
@@ -10,6 +10,10 @@ VERSION=$(toml-echo bitbox-bridge/Cargo.toml package.version)
 
 (
 	cd bitbox-bridge/release/darwin
-	pkgbuild --root tmp --scripts scripts --identifier ch.shiftcrypto.bitboxbridge --version "${VERSION}" --ownership recommended bridge.pkg
-	productbuild --distribution distribution.xml --resources resources --package-path . --version "${VERSION}" "${NAME}-${VERSION}-macOS-installer.pkg"
+	pkgbuild --root tmp --scripts scripts \
```

Comment:
Comment could be improved, but the scripts should be executed from the root project dir. They use paths relative to the working dir which won't work otherwise